bootstrap.js

```diff
@@ -6,19 +6,19 @@ const { exec } = require('child_process');
 
 // Array of folder names
 const folders = [
-  './',
-  'core',
-  'client',
-  'client-frameworks-support/client-support-angular',
-  'client-frameworks-support/testing-utilities',
-  'client-frameworks-support/client-support-ui5',
-  'container',
-  'plugins',
-  'scripts',
-  'test/e2e-test-application',
-  'test/e2e-test-application/externalMf',
-  'test/e2e-js-test-application',
-  'client-frameworks-support/testing-utilities/test'
+  // './',
```

Comment:
What's the point of keeping these lines commented? You may remove them as well as they're visible in git history anyway

Reply:
sorry, it was not mentioned . This is a DRAFT PR, mistakenly turned into a PR, so it is not yet ready for review.

Reply:
still in Progress. I will ping you when Ready 👍